.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -9,6 +9,15 @@ labels: kind/bug
 
 **What you expected to happen**:
 
+<!-- Please select an area, component, kind, platform and priority for this issue. This helps the community categorizing it. -->
+<!-- Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion. -->
```

Comment:
Should we add a comment stating that multiple categorizations are also possible? Like for example:
`/area/monitoring`
`/area/quality`